### PR TITLE
Improved App-native documentation based on feedback

### PR DESCRIPTION
**Pull request by himeshsiriwardana**
Improvement to multi-option and single-option content

Improved the reasons for how redirect mode is  implemented.

Added content on the **issuer** for native mode.

Added a limitation - Only supports silent JIT provisioning.

**Handle advanced login scenarios**

screencapture-localhost-8000-en-7-0-0-guides-authentication-app-native-authentication-handle-advanced-login-scenarios-2024-06-05-14_22_13

### CODE REVIEW

File: en/includes/guides/authentication/app-native-authentication/handle-advanced-login-scenarios.md [hunk 1]
```diff
@@ -29,19 +32,34 @@ To do so,
...
+
+!!! note "How is it different?"
+    
+    Although federated login flows in both [conventional login]({{base_path}}/guides/authentication/federated-login/) and app-native authentication redirect the user to the external IdP, there is a subtle difference between them.
+
+    - In a conventional flow, {{product_name}} redirects the user to the external IdP. Once the authentication is complete, the user is redirected back to {{product_name}} with an authorization code.
+
+    - In app-native authentication, {{product_name}} constructs a redirection URL (in which the callback URL is set to the application) and sends it to the application. The application redirects the user to the external IdP using the redirection URL. Once the authentication is complete, the IdP redirects the user back to the application (using the callback URL) with an authorization code. The application then sends the authorization code to {{product_name}}.
```
Comment (janakamarasena) [inline en/includes/guides/authentication/app-native-authentication/handle-advanced-login-scenarios.md:43]: Shall we add a bit more context here like `... in which the callback URL is set to the application using the callback URL sent in the initial authorization request)` IMO better to have this information so the developer will know exactly where the callback will be and be able to handle it in the app

**Comment by himeshsiriwardana** [comment]
Based on the feedback, I have updated the doc to be as follows.

screencapture-localhost-8000-en-7-0-0-guides-authentication-app-native-authentication-handle-advanced-login-scenarios-2024-06-06-16_12_31

File: en/includes/guides/authentication/app-native-authentication/handle-advanced-login-scenarios.md (hunk 1 above)
Comment (himeshsiriwardana) [reply en/includes/guides/authentication/app-native-authentication/handle-advanced-login-scenarios.md]: Ack. Please review the latest changes.

Comment (himeshsiriwardana) [reply en/includes/guides/authentication/app-native-authentication/handle-advanced-login-scenarios.md]: image

Comment (himeshsiriwardana) [reply en/includes/guides/authentication/app-native-authentication/handle-advanced-login-scenarios.md]: image

**Review by janakamarasena — APPROVED** [review]

**Pull request merged**